api/compose/v1alpha3/doc.go

```diff
@@ -0,0 +1,8 @@
+// Api versions allow the api contract for a resource to be changed while keeping
+// backward compatibility by support multiple concurrent versions
+// of the same resource
+
+// Package v1alpha3 is the current in dev version of the stack, containing evolution on top of v1beta2 structured spec
+// +k8s:openapi-gen=true
+// +k8s:conversion-gen=github.com/docker/compose-on-kubernetes/api/compose/v1beta2
```

Comment:
Shouldn't this be: `+k8s:conversion-gen=github.com/docker/compose-on-kubernetes/api/compose/v1alpha3`

Reply:
Nope, the idea was to generate automatic conversion from/to v1alpha3/v1beta2 (automatic conversion is a one shot process when you introduce a new API version, as at the beginning, they are identical. once you bring new fields in the structure, you must maintain those conversions by yourself)